src/views/EmbedView.vue

```diff
@@ -0,0 +1,44 @@
+<script setup>
+import { onBeforeMount, onMounted } from 'vue'
+import { useStore } from 'vuex'
+
+import I18nModule from '@/modules/i18n/I18nModule.vue'
+import InfoboxModule from '@/modules/infobox/InfoboxModule.vue'
+import MapFooter from '@/modules/map/components/footer/MapFooter.vue'
+import MapModule from '@/modules/map/MapModule.vue'
+import OpenFullAppLink from '@/utils/components/OpenFullAppLink.vue'
+import log from '@/utils/logging'
+
+const dispatcher = { dispatcher: 'EmbedView.vue' }
+
+const store = useStore()
+
+onBeforeMount(() => {
+    store.dispatch('setEmbeddedMode', { embed: true, ...dispatcher })
+})
+
+onMounted(() => {
+    log.info(`Map view mounted`)
+})
+</script>
+
+<template>
+    <div id="map-view">
+        <OpenFullAppLink />
+        <MapModule>
+            <InfoboxModule />
+        </MapModule>
+        <MapFooter />
+        <I18nModule />
+    </div>
+</template>
+
+<style lang="scss" scoped>
+@import 'src/scss/variables.scss';
+#map-view {
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    overflow: hidden;
+}
```

Comment:
might make sense to add this styling as a class `.map-view` declared in `webmapviewer-bootstrap-theme.scss` (or another specialized SCSS file) so that we may share how our "main view" are styled across the board

Reply:
Done